tornado/ioloop.py

```diff
@@ -254,10 +254,13 @@ def current(instance: bool = True) -> Optional["IOLoop"]:  # noqa: F811
         """
         try:
             loop = asyncio.get_event_loop()
-        except (RuntimeError, AssertionError):
+        except RuntimeError:
```

Comment:
Are you removing `AssertionError` here based on [this comment](https://github.com/tornadoweb/tornado/blob/9e9043e3cbf4605b272f74ede2c8e6a4d71049c1/tornado/platform/asyncio.py#L447) that says `AssertionError` was only used in Python 3.4?

Reply:
Yup, that's what I went on.